code/modules/recycling/disposal.dm

```diff
@@ -476,6 +479,14 @@
 				return
 
 			attackby(dropping, user)
+		else if(istype(dropping, /obj/structure/closet/crate) && can_load_crates())
```

Comment:
Also I just realized: it would be silly if it didn't work with wrapped crates so including /obj/item/delivery would be good.